api-docs/rst/dev-guide/concepts.rst

```diff
@@ -217,6 +254,51 @@ servers.
  .. _List Load Balancer Details: http://docs.rackspace.com/loadbalancers/api/v1.0/clb-getting-started/content/List_LB_Details.html
 
 
+.. _stack-parameters:
+
+Stack parameters
+^^^^^^^^^^^^^^^^
+
+Stack parameters specify the configuration of the stacks that will be
+created during the scaling process. Note the ``stack`` arguments are
+directly passed to Cloud Orchestration when creating a stack. For more
+information, see
+`Create Stack<http://api.rackspace.com/api-ref-orchestration.html#stack_create>`__.
+
+``template``
+    The template that describes the stack. Either template or
+    template_url must be specified. See `Create
```

Comment:
Not sure if "Either template or template_url must be specified" makes sense here.  Shouldn't it just be a template, since there's a separate option for template_url?